src/applications/disability-benefits/new-686/config/chapters/veteran-information/veteran-address/veteran-address.js

```diff
@@ -0,0 +1,52 @@
+import { genericSchemas } from '../../../generic-schema';
+import emailUI from 'platform/forms-system/src/js/definitions/email';
+
+export const schema = {
+  type: 'object',
+  required: [],
+  properties: {
+    countryDropdown: genericSchemas.countryDropdown,
+    street: genericSchemas.genericTextinput,
+    line2: genericSchemas.genericTextinput,
+    line3: genericSchemas.genericTextinput,
+    city: genericSchemas.genericTextinput,
+    postalCode: genericSchemas.genericNumberInput,
+    phoneNumber: genericSchemas.genericNumberInput,
+    emailAddress: genericSchemas.genericTextinput,
```

Comment:
I think email address should by something like `{type: 'string', format: 'email'}`. In this case it'd probably be better to use json-schema's built-in format - [more here](https://json-schema.org/understanding-json-schema/reference/string.html#format). Using the generic text regex here actually causes a validation error on the email field.

Reply:
Agreed. I added an email field to the generic schema and used that.